applications/FluidDynamicsApplication/tests/test_fluid_auxiliary_utilities.py

```diff
@@ -113,6 +113,39 @@ def testCalculateFlowRateNegativeSkin(self):
         negative_flow_rate_top = KratosFluid.FluidAuxiliaryUtilities.CalculateFlowRateNegativeSkin(top_skin_model_part)
         self.assertAlmostEqual(negative_flow_rate_top, ref_negative_flow_rate_top, 12)
 
+    def testCalculateFluidVolume(self):
+        fluid_model_part = self.model.GetModelPart("FluidModelPart")
+        fluid_model_part.AddNodalSolutionStepVariable(Kratos.DISTANCE)
```

Comment:
You are adding a nodal variable to a model part which is already populated with nodes.

Reply:
ok, anyway the distance was there so it was not needed